roles/softserve/templates/application.cfg

```diff
@@ -3,7 +3,8 @@ SQLALCHEMY_DATABASE_URI = 'postgresql://{{ softserve_config['db_user'] }}:{{ sof
 SECRET_KEY = '{{ softserve_config['app_secret_key'] }}'
 AWS_ACCESS_KEY_ID = '{{ softserve_config['aws_access_key_id'] }}'
 AWS_SECRET_ACCESS_KEY = '{{ softserve_config['aws_secret_access_key'] }}'
-IMAGE_ID = '{{ softserve_config['image_id'] }}'
```

Comment:
I think I would keep also the old variable here with a comment, since this would ease any code migration (like if we have to revert, we would still use the same file).

Also, do we really need to image_id to be variable ? They are not secret, so we could just put them directly here and avoid using the private repo.

Reply:
Yeah, they're not secret. I will update it here only

Reply:
If we deploy that without the softserve modification, wouldn't it break ?

Reply:
I have merged the changes in softserve https://github.com/gluster/softserve/pull/61